### ISSUE TITLE
Planning our testing & coverage

### ISSUE THREAD

**Issue by TheVulch** [Planteome/planteome-data-visualization#8]
There (will be) alot of code conducting computation - something we should definitely test. What test method/framework/thoroughness will we use for this project?

**Comment by qubotong** [comment]
As I said last time, we could use AgriGO to confirm our computation results. With using customized input and customized output, we should get same p-value from AgriGO and our web.

**Comment by TheVulch** [comment]
yes, we can definitely test our statistical methods with AgriGO (we probably have edge cases for the stat methods, though)
Also, we have alot of other code that's not just for the statistics (DOM handling/etc.). We might not need to set up a testing strategy now, but it'll get more and more useful as we add complex code (visualization) to the project. I'd say we should just leave this open for when we get to visualization.
